test-cases/tests/alerts/c16-verify-ocm-sendgrid-service-alerts.md

```diff
@@ -0,0 +1,25 @@
+---
+environments:
+  - osd-post-upgrade
+estimate: 30m
+tags:
+  - per-release
+---
+
+# C16 - Verify OCM SendGrid Service alerts
+
+## Description
+
+This test should verify that the expected alerts for the OCM SendGrid Service exist in the OCM Prometheus instance.
+
+## Steps
+
+[List here the the steps that the tester need to perform, the expected result of the step is quoted after the step]
+
+1. Go to the [OCM staging Prometheus instance](https://prometheus.app-sre-stage-01.devshift.net/alerts)
+
+2. Login via the OpenShift option, using the `github-app-sre` identity provider
+
+   - If you do not have permissions via this identity provider, contact the `sd-app-sre` Slack channel
+
+3. Ensure each alert defined in [the alert definition](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/resources/observability/prometheusrules/ocm-sendgrid-svc-stage.prometheusrules.yaml) exists in the list of Prometheus alerts, and none are firing
```

Comment:
i still have to:

1. confirm this will be the final route to the staging alerts
2. include information about the production alerts

Reply:
Confirmed with App SRE that we will have access to the production prometheus to check the rules.

As the alerts will only be added to prod in https://issues.redhat.com/browse/INTLY-9830 I'd be tempted to update that ticket to make a minor update to this SOP to perform the same set of checks for production once the alerts have been added there.

That'd mean this can be merged once ocm-sendgrid-svc-stage.prometheusrules.yaml has been merged into app interface.

@pmccarthy @laurafitzgerald Does this sound ok to you?